cernopendata/modules/records/serializers/schemas/schemaorg.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Open Data Portal.
+# Copyright (C) 2018 CERN.
+#
+# CERN Open Data Portal is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# CERN Open Data Portal is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with CERN Open Data Portal; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+"""CERN Opendata schema.org marshmallow schema."""
+
+from __future__ import absolute_import, print_function
+
+from marshmallow import Schema, fields
+
+
+class DatasetSchemaorgSchema(Schema):
+    """schema.org JSON-LD schema for CERN Opendata Dataset Record."""
+
+    # TODO: Implement schema.org structure and keys
+
+    type = fields.Method('get_type')
+
+    def get_type(self, obj):
```

Comment:
I don't really get the use of get_type as here https://github.com/cernopendata/opendata.cern.ch/pull/2277/files#diff-88d5e62a55ecff4c110e58decd763ce6R55 you check the type directly.

Reply:
Sorry, I force pushed new commit before I had a look.